crates/node_binding/src/plugins/interceptor.rs

```diff
@@ -681,6 +684,14 @@ define_register!(
   kind = RegisterJsTapKind::CompilationAfterProcessAssets,
   skip = true,
 );
+define_register!(
+  RegisterCompilationSealTaps,
+  tap = CompilationSealTap<(), ()> @ CompilationSealHook,
+  cache = false,
+  sync = true,
```

Comment:
`sync` means at rust side you can't call it in async way, basically only `executeModule` need `sync: true`